packages/dds/merge-tree/src/attributionCollection.ts

```diff
@@ -390,20 +395,23 @@ export class AttributionCollection implements IAttributionCollection<Attribution
 	): void {
 		const { channels } = summary;
 		assert(
+			// eslint-disable-next-line unicorn/consistent-destructuring
```

Comment:
Is there cost to accessing `seqs`? Curious why we need to disable this. If we need to, we should add a comment explaining why.